Index: src/main/java/com/lordFenix56/learningMod/proxy/ClientProxy.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- src/main/java/com/lordFenix56/learningMod/proxy/ClientProxy.java	(revision )
+++ src/main/java/com/lordFenix56/learningMod/proxy/ClientProxy.java	(revision )
@@ -0,0 +1,7 @@
+package com.lordFenix56.learningMod.proxy;
+
+/**
+ * Created by Leandro on 20/10/2014.
+ */
+public class ClientProxy extends CommonProxy {
+}
Index: src/main/java/com/lordFenix56/learningMod/proxy/IProxy.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- src/main/java/com/lordFenix56/learningMod/proxy/IProxy.java	(revision )
+++ src/main/java/com/lordFenix56/learningMod/proxy/IProxy.java	(revision )
@@ -0,0 +1,7 @@
+package com.lordFenix56.learningMod.proxy;
+
+/**
+ * Created by Leandro on 20/10/2014.
+ */
+public interface IProxy {
+}
Index: src/main/java/com/lordFenix56/learningMod/reference/Reference.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- src/main/java/com/lordFenix56/learningMod/reference/Reference.java	(revision )
+++ src/main/java/com/lordFenix56/learningMod/reference/Reference.java	(revision )
@@ -0,0 +1,13 @@
+package com.lordFenix56.learningMod.reference;
+
+/**
+ * Created by Leandro on 20/10/2014.
+ */
+public class Reference {
+    public static final String MOD_ID ="LearningMod";
+    public static final String MOD_NAME ="Learning Mod";
+    public static final String VERSION ="1.7.10-0.1";
+    public static final String CLIENT_PROXY_CLASS= "com.lordFenix56.learningMod.proxy.ClientProxy";
+    public static final String SERVER_PROXY_CLASS="com.lordFenix56.learningMod.proxy.ServerProxy";
+
+}
Index: src/main/java/com/lordFenix56/learningMod/LearningMod.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- src/main/java/com/lordFenix56/learningMod/LearningMod.java	(revision )
+++ src/main/java/com/lordFenix56/learningMod/LearningMod.java	(revision )
@@ -0,0 +1,39 @@
+package com.lordFenix56.learningMod;
+
+import com.lordFenix56.learningMod.proxy.IProxy;
+import com.lordFenix56.learningMod.reference.Reference;
+import cpw.mods.fml.common.Mod;
+import cpw.mods.fml.common.SidedProxy;
+import cpw.mods.fml.common.event.FMLInitializationEvent;
+import cpw.mods.fml.common.event.FMLPostInitializationEvent;
+import cpw.mods.fml.common.event.FMLPreInitializationEvent;
+
+/**
+ * Created by Leandro on 20/10/2014.
+ */
+
+@Mod(modid= Reference.MOD_ID, name=Reference.MOD_NAME, version= Reference.VERSION)
+public class LearningMod {
+    @Mod.Instance(Reference.MOD_ID)
+    public static LearningMod instance;
+
+    @SidedProxy(clientSide = Reference.CLIENT_PROXY_CLASS, serverSide = Reference.SERVER_PROXY_CLASS)
+    public static IProxy proxy;
+
+    @Mod.EventHandler
+    public void preInit(FMLPreInitializationEvent event)
+    {
+
+    }
+    @Mod.EventHandler
+    public void init (FMLInitializationEvent event)
+    {
+
+    }
+    @Mod.EventHandler
+    public void postInit(FMLPostInitializationEvent event)
+    {
+
+    }
+
+}
Index: build.gradle
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- build.gradle	(revision )
+++ build.gradle	(revision )
@@ -0,0 +1,63 @@
+buildscript {
+    repositories {
+        mavenCentral()
+        maven {
+            name = "forge"
+            url = "http://files.minecraftforge.net/maven"
+        }
+        maven {
+            name = "sonatype"
+            url = "https://oss.sonatype.org/content/repositories/snapshots/"
+        }
+    }
+    dependencies {
+        classpath 'net.minecraftforge.gradle:ForgeGradle:1.2-SNAPSHOT'
+    }
+}
+
+apply plugin: 'forge'
+
+version = "1.7.10-0.1"
+group= "com.lordFenix56.learningMod" // http://maven.apache.org/guides/mini/guide-naming-conventions.html
+archivesBaseName = "learningMod"
+
+minecraft {
+    version = "1.7.10-10.13.2.1231"
+    runDir = "eclipse"
+}
+
+dependencies {
+    // you may put jars on which you depend on in ./libs
+    // or you may define them like so..
+    //compile "some.group:artifact:version:classifier"
+    //compile "some.group:artifact:version"
+      
+    // real examples
+    //compile 'com.mod-buildcraft:buildcraft:6.0.8:dev'  // adds buildcraft to the dev env
+    //compile 'com.googlecode.efficient-java-matrix-library:ejml:0.24' // adds ejml to the dev env
+
+    // for more info...
+    // http://www.gradle.org/docs/current/userguide/artifact_dependencies_tutorial.html
+    // http://www.gradle.org/docs/current/userguide/dependency_management.html
+
+}
+
+processResources
+{
+    // this will ensure that this task is redone when the versions change.
+    inputs.property "version", project.version
+    inputs.property "mcversion", project.minecraft.version
+
+    // replace stuff in mcmod.info, nothing else
+    from(sourceSets.main.resources.srcDirs) {
+        include 'mcmod.info'
+                
+        // replace version and mcversion
+        expand 'version':project.version, 'mcversion':project.minecraft.version
+    }
+        
+    // copy everything else, thats not the mcmod.info
+    from(sourceSets.main.resources.srcDirs) {
+        exclude 'mcmod.info'
+    }
+}
Index: src/main/java/com/lordFenix56/learningMod/proxy/CommonProxy.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- src/main/java/com/lordFenix56/learningMod/proxy/CommonProxy.java	(revision )
+++ src/main/java/com/lordFenix56/learningMod/proxy/CommonProxy.java	(revision )
@@ -0,0 +1,7 @@
+package com.lordFenix56.learningMod.proxy;
+
+/**
+ * Created by Leandro on 20/10/2014.
+ */
+public abstract class CommonProxy implements IProxy {
+}
Index: src/main/resources/mcmod.info
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- src/main/resources/mcmod.info	(revision )
+++ src/main/resources/mcmod.info	(revision )
@@ -0,0 +1,17 @@
+[
+{
+  "modid": "LearningMod",
+  "name": "Learning Mod",
+  "description": "I'm learning to mod",
+  "version": "0.1",
+  "credits": "Created by LordFenix56",
+  "logoFile": "",
+  "mcversion": "1.7.10",
+  "url": "",
+  "updateUrl": "",
+  "authors": ["LordFenix56"],
+  "parent":"",
+  "screenshots": [],
+  "dependencies": []
+}
+]
\ No newline at end of file
Index: src/main/java/com/lordFenix56/learningMod/configuration/ConfigurationHandle.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- src/main/java/com/lordFenix56/learningMod/configuration/ConfigurationHandle.java	(revision )
+++ src/main/java/com/lordFenix56/learningMod/configuration/ConfigurationHandle.java	(revision )
@@ -0,0 +1,7 @@
+package com.lordFenix56.learningMod.configuration;
+
+/**
+ * Created by Leandro on 31/10/2014.
+ */
+public class ConfigurationHandle {
+}
Index: src/main/java/com/lordFenix56/learningMod/proxy/ServerProxy.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- src/main/java/com/lordFenix56/learningMod/proxy/ServerProxy.java	(revision )
+++ src/main/java/com/lordFenix56/learningMod/proxy/ServerProxy.java	(revision )
@@ -0,0 +1,7 @@
+package com.lordFenix56.learningMod.proxy;
+
+/**
+ * Created by Leandro on 20/10/2014.
+ */
+public class ServerProxy extends CommonProxy{
+}
